Buy/Generated/Storefront/Shop.swift

```diff
@@ -297,8 +297,8 @@ extension Storefront {
 		/// - parameters:
 		///     - first: No description
```

Comment:
Are these generated? missing `alias` parameter comment

Reply:
Yeah, this is expected behaviour.